### PR TITLE
fixes .exe misconception

### PR DESCRIPTION
**Pull request by lucaszanella**
fixes misconception about .exe being for Windows executables

### CODE REVIEW

**Comment by jeremiedimino** [comment]
Thanks. The message feels a tad too prominent to me, it should be a note after the example. Perhaps in a smaller font  or in italic.

**Comment by lucaszanella** [comment]
Take a look now

File: index.md [hunk 1]
```diff
@@ -36,11 +36,14 @@ Lwt_main.run (Lwt_io.printf "Hello, world!\n")
 ```
 
 #### Your shell
+
 ```
 $ dune build hello_world.exe
 $ ./_build/default/hello_world.exe
 Hello, world!
 ```
+*To compile into native native mode, we put a .exe in front of the name. This does not have anything to do with Windows executables. For bytecode compilation we put .bc in the end.*
```
Comment (jeremiedimino) [inline index.md:45]:
```suggestion

*Note: Dune uniformly uses the `.exe` extension to build native executables, even on Unix where programs don't usually have a `.exe` extension.*
```

**Comment by jeremiedimino** [comment]
Thanks, I reworded the note and merged

**Pull request merged**